rozy/src/main.rs

```diff
@@ -452,6 +459,10 @@ fn main() -> Result<(), Error> {
 
     if invoked_as.starts_with("ozy") {
         let args = Args::parse();
+        if !did_path_contain_ozy {
+            show_path_warning()?;
```

Comment:
This will print to stdout when using `ozy run` to invoke an app, which is unacceptable.

This warning should only be printed when running `ozy init`, like before.